### ISSUE TITLE
Integrate with new dynamic plugin system

### ISSUE THREAD

**Issue by KraigM** [KraigM/homebridge-harmonyhub#28]
HomeBridge now supports dynamic plugins (v >= 3) which should allow us to add and remove hubs as they appear/disappear from the network.  If we integrate with this functionality properly, we should be able to solve a lot of the connection issues as well as multiple hubs support.

**Comment by KraigM** [comment]
We will need to make sure that if a hub goes offline and then comes back online, that we don't lose the HomeKit settings

**Comment by leoneleone** [comment]
yes please. I'm experiencing some stability issues with the harmonyhub plugin (and you're homebridge-nest plugin incidentally) since updating Homebridge to (v >= 3).

Is there any current way to promote a stable initialisation of the plugin in homebridge?

**Comment by KraigM** [comment]
This is implemented in the upcoming version (0.2.0). You can try out the next version (keeping in mind that it is a prerelease) by using the upgrade command:
`npm install -g homebridge-harmonyhub@0.2.0-alpha`
